src/index.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import payment from 'payment';
 import creditCardType from 'credit-card-type';
 import styled from 'styled-components';
+import moment from 'moment';
```

Comment:
As much as I love moment, it's bundle size is off the charts (310.5K), which is a hefty price since this package is used on the web. The end user will have to download an extra 310.5K to use this. Which could take ~3 seconds to load on a 3G connection. 

What if we use [date-fns](https://date-fns.org/) instead. For our case, we would use [isValid(date)](https://date-fns.org/v1.29.0/docs/isValid).

Reply:
🆗 ill look into it tomorrow, didn't realise moment was so large!